pyproject.toml

```diff
@@ -30,7 +30,7 @@ reportMissingTypeStubs = "warning"
 reportOptionalMemberAccess = "warning"
 reportUnboundVariable = "warning"
 reportPrivateImportUsage = "warning"
-reportTypedDictNotRequiredAccess = "warning"
+reportTypedDictNotRequiredAccess = "none"
```

Comment:
If you think any issues may lurk here, open a follow-up ticket to elevate this back to `error`

Reply:
No, it just produces such stuff due to non-optimal type definitions in dependencies:
```
  /var/lib/buildkite-agent/builds/buildkite-15f2293-i-01d2a48196187df78-1/materialize/tests/ci/cleanup/aws.py:44:16 - warning: Could not access item in TypedDict
    "Name" is not a required key in "BucketTypeDef", so access may result in runtime exception (reportTypedDictNotRequiredAccess)
```

Reply:
ok, silence it then, thanks!